### PR TITLE
Some quality of life improvements

### PR DESCRIPTION
**Pull request by matthewkennedy5**
* Only download the StyleTTS2 models from S3, not everything in the prototype folder
* Set all observability logs to be shown by default so you don't have to click a bunch of buttons
* Increase timeout to 5 mins so it doesn't close on you during testing

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ FROM nvcr.io/nvidia/pytorch:24.01-py3
 ARG DEBIAN_FRONTEND=noninteractive
 WORKDIR /openduck-py
 
-COPY ./openduck-py /openduck-py
+# Copying is only necessary on prod, when you're not mounting the volume
+# COPY ./openduck-py /openduck-py
```
Comment (zachwe) [inline Dockerfile:6]: do you know if CI deployments are working and if this will break them?

**Review by zachwe — APPROVED** [review]
LGTM % a question

File: Dockerfile (hunk 1 above)
Comment (matthewkennedy5) [reply Dockerfile:6]: oh not sure

File: docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ services:
   web:
     image: 613148481050.dkr.ecr.us-west-2.amazonaws.com/openduck:latest
     build: .
-    command: bash -c "python setup.py develop && aws s3 cp --recursive s3://uberduck-models-us-west-2/prototype models && uvicorn openduck_py.routers.main:app --reload --host 0.0.0.0 --port 8000"
+    command: bash -c "python setup.py develop && aws s3 cp --recursive s3://uberduck-models-us-west-2/prototype/download models && uvicorn openduck_py.routers.main:app --reload --host 0.0.0.0 --port 8000"
```
Comment (Sobsz) [inline docker-compose.yml:6]: i am once again asking if `aws s3 sync` wouldn't be better

Comment (matthewkennedy5) [reply docker-compose.yml:6]: so true

**Pull request merged**